warning: ‘const QRect QDesktopWidget::screenGeometry(int) const’ is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp: In constructor ‘OpenMS::TOPPViewBase::TOPPViewBase(OpenMS::TOPPViewBase::TOOL_SCAN, QWidget*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp:132:68: warning: ‘const QRect QDesktopWidget::screenGeometry(int) const’ is deprecated: Use QGuiApplication::screens() [-Wdeprecated-declarations]
  132 |     QRect screen_geometry = QApplication::desktop()->screenGeometry();
      |                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
In file included from /usr/include/qt/QtWidgets/QtWidgets:58,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/PlotCanvas.h:48,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h:48,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp:35:
/usr/include/qt/QtWidgets/qdesktopwidget.h:79:67: note: declared here
   79 |     QT_DEPRECATED_X("Use QGuiApplication::screens()") const QRect screenGeometry(int screen = -1) const;
      |                                                                   ^~~~~~~~~~~~~~
```

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp
@@ -88,7 +88,7 @@
 #include <QtCore/QDir>
 #include <QtCore/QSettings>
 #include <QtCore/QUrl>
-#include <QtWidgets/QDesktopWidget>
+#include <QGuiApplication>
 #include <QtWidgets/QDockWidget>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QMessageBox>
@@ -129,7 +129,7 @@ namespace OpenMS
     setAcceptDrops(true); // enable drag-and-drop
 
     // get geometry of first screen
-    QRect screen_geometry = QApplication::desktop()->screenGeometry();
+    QRect screen_geometry = QGuiApplication::primaryScreen()->geometry();
     // center main window
     setGeometry(
       (int)(0.1 * screen_geometry.width()),

